yarn.lock

```diff
@@ -11090,16 +11090,11 @@ preserve@^0.2.0:
   resolved "https://registry.yarnpkg.com/preserve/-/preserve-0.2.0.tgz#815ed1f6ebc65926f865b310c0713bcb3315ce4b"
   integrity sha1-gV7R9uvGWSb4ZbMQwHE7yzMVzks=
 
-prettier@2.2.1:
+prettier@2.2.1, "prettier@>=2.2.1 <=2.3.0":
```

Comment:
didnt want to update to the newest?

Reply:
If I did, then it would change all the files and I would also need to force resolve because this `>=2.2.1 <=2.3.0` comes from storybook.